index.html

```diff
@@ -532,6 +532,19 @@ <h3 id="navigation-scope-security-considerations">
           security reasons. It ensures that users are always aware of which
           <a>origin</a> they are interacting with.
         </p>
+        <p>
+          Despite this, there is still a potential spoofing risk, if an
+          installed app pretends to navigate to an out-of-scope site on another
+          <a>origin</a>. The site shows a fake version of the user agent's
```

Comment:
Very long sentences, can it be split up to make it more readable?

Reply:
Done!